<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口监视器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="picture.qrc">
    <normaloff>:/pic/app_logo.png</normaloff>:/pic/app_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="message"/>
          </item>
          <item>
           <widget class="QPushButton" name="Send_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
            <property name="icon">
             <iconset resource="picture.qrc">
              <normaloff>:/pic/send.png</normaloff>:/pic/send.png</iconset>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="adjustable_splitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTextBrowser" name="msg">
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="1">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>校验位</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QComboBox" name="set_flow_ctrl">
                <item>
                 <property name="text">
                  <string>无流控</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>硬件流控</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>软件流控</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QComboBox" name="set_parity_bits">
                <item>
                 <property name="text">
                  <string>无校验位</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>偶校验</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>奇校验</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>校验位始终为0</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>校验位始终为1</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QComboBox" name="set_stop_bits">
                <item>
                 <property name="text">
                  <string>1位</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1.5位</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2位</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>停止位</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QComboBox" name="set_data_bits">
                <item>
                 <property name="text">
                  <string>5位</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>6位</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>7位</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8位</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>流控</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>数据位</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>发送记录(双击填充到消息发送框)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="msg_history"/>
            </item>
            <item>
             <widget class="QCheckBox" name="duplicate_allowed">
              <property name="text">
               <string>不添加重复项</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="operate_port_btn">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="clear_aftersend">
            <property name="text">
             <string>发送完成后清空</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="is_send_return">
            <property name="text">
             <string>发送回车</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="isTop">
            <property name="text">
             <string>置顶窗口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>显示字号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="font_size">
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="port_list"/>
          </item>
          <item>
           <widget class="QPushButton" name="refrush">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;刷新串口&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="picture.qrc">
              <normaloff>:/pic/refrush.png</normaloff>:/pic/refrush.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>波特率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="BaudRate"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="isShowTime">
            <property name="text">
             <string>显示时间</string>
            </property>
            <property name="tristate">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clear">
            <property name="text">
             <string>清空输出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="operate_port"/>
    <addaction name="save_msg"/>
    <addaction name="clear_history"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="save_msg">
   <property name="icon">
    <iconset resource="picture.qrc">
     <normaloff>:/pic/save.png</normaloff>:/pic/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="picture.qrc">
     <normaloff>:/pic/exit.png</normaloff>:/pic/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="operate_port">
   <property name="text">
    <string>_</string>
   </property>
  </action>
  <action name="clear_history">
   <property name="icon">
    <iconset resource="picture.qrc">
     <normaloff>:/pic/history.png</normaloff>:/pic/history.png</iconset>
   </property>
   <property name="text">
    <string>清除发送记录</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>msg</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_history</sender>
   <signal>triggered()</signal>
   <receiver>msg_history</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
